src/uiSelectController.js

```diff
@@ -375,7 +375,11 @@ uis.controller('uiSelectCtrl',
       if ( ! ctrl.items && ! ctrl.search && ! ctrl.tagging.isActivated) return;
 
       if (!item || !_isItemDisabled(item)) {
-        if(ctrl.tagging.isActivated) {
+        // if click is made on existing item, prevent from tagging, ctrl.search does not matter
+        var manualSelection = false;
+        if($event && $event.type === 'click' && item)
```

Comment:
We have a check down below already...
```js
if ($event && $event.type === 'click') {
   ctrl.clickTriggeredSelect = true;
 }
```
Can you combine this/assignment into one and then just check for item?